assets/src/edit-story/components/library/panes/pageLayouts/pageLayouts.js

```diff
@@ -159,6 +184,30 @@ function PageLayouts({ pages, parentRef }) {
         height: pageSize.height,
       }}
     >
+      <ActionRow>
+        <Headline
+          as="h3"
+          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XXX_SMALL}
+        >
+          {__('Layouts', 'web-stories')}
+        </Headline>
+        <LayoutsToggle>
+          <Text
+            as="label"
+            htmlFor={toggleId}
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+          >
+            {__('Show Images', 'web-stories')}
+          </Text>
+          <Toggle
+            id={toggleId}
+            aria-label={__('Show images in layouts', 'web-stories')}
```

Comment:
Do we like this aria label for the toggle?

Reply:
Thinking about this again - it may not be needed since the toggle has a `label` that's associated with it